test/libsolidity/util/SoltestTypes.h

```diff
@@ -125,6 +126,8 @@ struct ABIType
 	Type type = ABIType::None;
 	Align align = ABIType::AlignRight;
 	size_t size = 32;
+	size_t M = 0;
```

Comment:
Please use `bits` and `fractionalDigits`. Do we actually need the "bits" part? It's not needed for integers, so it should also not be needed for fixed point types.

Reply:
Right now I use the `bits` part it in the formatting function to check for potential overflows. I think that would be important to have, so we can be sure that the function parameters where supplied correctly.

Reply:
I just realised that right now negative values are also not yet implemented for `fixed`. I think because of this, we will need the `bits` part to set the value for negative values correctly.

Reply:
the same applies for negative integer values, please keep it as simple as possible.